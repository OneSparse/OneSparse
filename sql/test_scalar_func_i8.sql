set client_min_messages = 'WARNING';
create extension if not exists onesparse;
set search_path = 'onesparse';
select '-1'::bigint::scalar;
select '0'::bigint::scalar;
select '1'::bigint::scalar;
