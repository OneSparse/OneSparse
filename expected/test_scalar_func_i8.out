set client_min_messages = 'WARNING';
create extension if not exists onesparse;
set search_path = 'onesparse';
select '-1'::bigint::scalar;
 scalar 
--------
 i8:-1
(1 row)

select '0'::bigint::scalar;
 scalar 
--------
 i8:0
(1 row)

select '1'::bigint::scalar;
 scalar 
--------
 i8:1
(1 row)

select set('1'::bigint::scalar, 2);
 set  
------
 i8:2
(1 row)

select scalar_bigint((-1)::bigint);
 scalar_bigint 
---------------
 i8:-1
(1 row)

select scalar_bigint((0)::bigint);
 scalar_bigint 
---------------
 i8:0
(1 row)

select scalar_bigint((1)::bigint);
 scalar_bigint 
---------------
 i8:1
(1 row)

select bigint_scalar((-1)::bigint::scalar);
 bigint_scalar 
---------------
            -1
(1 row)

select bigint_scalar((0)::bigint::scalar);
 bigint_scalar 
---------------
             0
(1 row)

select bigint_scalar((1)::bigint::scalar);
 bigint_scalar 
---------------
             1
(1 row)

